docs/reference/search/profile.asciidoc

```diff
@@ -276,7 +282,7 @@ the equivalent name in Elasticsearch.  The `"description"` field displays the Lu
 is made available to help differentiating between parts of your query (e.g. both `"message:search"` and `"message:test"`
 are TermQuery's and would appear identical otherwise.
 
-The `"time"` field shows that this query took ~15ms for the entire BooleanQuery to execute.  The recorded time is inclusive
+The `"time_in_nanos"` field shows that this query took ~1.8ms for the entire BooleanQuery to execute.  The recorded time is inclusive
```

Comment:
I thing the 15ms was an error, the example above used to have "1.873811000ms" in its main node output, so I changed this accordingly. Maybe @polyfractal can confirm this as an existing error in the old docs.